src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form_group">
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          How old are You?
+          <input
+            type="number"
+            name="Age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form_field">
+          Full date of birth:
+          <input type="date" name="Date">
+        </label>
+
+        <label class="form_field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="Agreed"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form_group">
+        <legend>Registration</legend>
+        <label class="form_field">
+          E-mail:
+          <input
+            type="email"
+            name="Email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form_field">
+          Password:
+          <input
+            type="password"
+            name="Password"
+            minlength="3"
+            maxlength="32"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form_group">
+        <legend>An interesting fact about you!</legend>
+        <div class="form_field">
+          Do you love cats?
+          <input
+            type="radio"
+            name="Cats"
+            value="Y"
+            id="radio-y"
+          >
+          <label for="radio-y">Yes</label>
+          <input
+            type="radio"
+            name="Cats"
+            value="N"
+            id="radio-n"
+          >
+          <label for="radio-n">No</label>
+        </div>
+
+        <label class="form_field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+
+        <label class="form_field">
+          What time do you go to bed?
+          <input type="time" name="Time">
+        </label>
+
+        <label class="form_field">
+          What are your favorite brand of cars?
+          <select name="Car" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Laba">Lada</option>
+            <option value="Brand" selected>
```

Comment:
What is that?
![2023-03-21_13h51_14](https://user-images.githubusercontent.com/61904995/226598034-51f7c3d9-cc41-4755-8947-02a6279502f8.png)